index.html

```diff
@@ -195,90 +203,95 @@ <h2 class="works-card-title">Multi-Post Stories</h2>
+                <p class="about-lang-name">CSS</p>
+              </li>
+            </ul>
           </li>
-          <li class="about-icons-li">
-            <a href="#" class="about-icons-li-a"
-              ><img src="/assets/linkedin.svg.svg" alt="github icon"
-            /></a>
+          <li class="about-tool border-bottom">
+            <div class="about-languages-list">
+              <h3 class="about-lang-title">Frameworks</h3>
+              <img
+                class="about-lang-img"
+                src="./assets/chev-right.png"
+                alt="arrow icon"
+              />
+            </div>
           </li>
-          <li class="about-icons-li">
-            <a href="#" class="about-icons-li-a"
-              ><img src="/assets/medium.svg.svg" alt="github icon"
-            /></a>
+          <li class="about-tool border-bottom">
+            <div class="about-languages-list">
+              <h3 class="about-lang-title">Skills</h3>
+              <img
+                class="about-lang-img"
+                src="./assets/chev-right.png"
+                alt="arrow icon"
+              />
+            </div>
           </li>
         </ul>
       </div>
-      <ul class="about-tools-list">
-        <li class="about-tool">
-          <div class="about-languages-list">
-            <h3 class="about-lang-title">Languages</h3>
-            <img
-              class="about-lang-img"
-              src="./assets/chev-down.png"
-              alt="arrow icon"
-            />
-          </div>
-          <ul class="about-lang">
-            <li class="about-lang-row">
-              <img src="./assets/javascript.svg.svg" alt="javascript icon" />
-              <p class="about-lang-name">JavaScript</p>
-            </li>
-            <li class="about-lang-row">
-              <img src="./assets/html.svg.svg" alt="javascript icon" />
-              <p class="about-lang-name">HTML</p>
-            </li>
-            <li class="about-lang-row">
-              <img src="./assets/css.svg.svg" alt="javascript icon" />
-              <p class="about-lang-name">CSS</p>
-            </li>
-          </ul>
-        </li>
-        <li class="about-tool border-bottom">
-          <div class="about-languages-list">
-            <h3 class="about-lang-title">Frameworks</h3>
-            <img
-              class="about-lang-img"
-              src="./assets/chev-right.png"
-              alt="arrow icon"
-            />
-          </div>
-        </li>
-        <li class="about-tool border-bottom">
-          <div class="about-languages-list">
-            <h3 class="about-lang-title">Skills</h3>
-            <img
-              class="about-lang-img"
-              src="./assets/chev-right.png"
-              alt="arrow icon"
-            />
-          </div>
-        </li>
-      </ul>
     </section>
     <footer class="Contact-form">
       <div class="contact_container">
```

Comment:
- [ ] Great job with the footer section 💯 but it still lacks some background images. It would be great to add those so as to improve the user interface.
- [ ] Also your `Get in touch` button is not aligned in the middle as shown in the template. Kindly address this issue.
- [ ] It would be great to implement the button interactions for all the buttons on your site for a better user experience 👍 
- Your design:
![Annotation 2022-02-26 102345](https://user-images.githubusercontent.com/85113567/155838337-c64ab1b5-a2bd-4f92-894b-20c16e2ec0c1.jpg)
- Figma design:
![Annotation 2022-02-26 102357](https://user-images.githubusercontent.com/85113567/155838342-9d18660e-7281-489b-8ef8-9b743c3bb9db.jpg)

Reply:
* Added button interactions.

Reply:
* Issue resolved.